service/service.py

```diff
@@ -140,45 +163,62 @@ def delete(self, rec_id):
         if recommendation:
             recommendation.delete()
         return '', status.HTTP_204_NO_CONTENT
+		
+    #------------------------------------------------------------------
+    # RETRIEVE A RECOMMENDATION
+    #------------------------------------------------------------------
+    @api.doc('get_recommendations')
+    @api.response(404, 'Recommendation not found')
+    @api.marshal_with(recommendation_model)
+    def get(self, rec_id):
+        """
+        Retrieve a single Recommendation
+        This endpoint will return a Recommendation based on it's id
+        """
+        app.logger.info("Request to Retrieve a recommendation with id [%s]", rec_id)
+        recommendation = Recommendation.find(rec_id)
+        if not recommendation:
+            api.abort(status.HTTP_404_NOT_FOUND, "Recommendation with id '{}' was not found.".format(rec_id))
+        return recommendation.serialize(), status.HTTP_200_OK
 
-
+		
 # #####################################################################
 # RETRIEVE A RECOMMENDATION
 # #####################################################################
-@app.route('/recommendations/<int:rec_id>', methods=['GET'])
-def get_recommendations(rec_id):
-    """
-    Retrieve a single Recommendation
-    This endpoint will return a Recommendation based on it's id
-    """
-    app.logger.info('Request for recommendation with id: %s', rec_id)
-    recommendation = Recommendation.find(rec_id)
-    if not recommendation:
-        raise NotFound("Recommendation with id '{}' was not found.".format(rec_id))
-    return make_response(jsonify(recommendation.serialize()), status.HTTP_200_OK)
-
+#@app.route('/recommendations/<int:rec_id>', methods=['GET'])
```

Comment:
you can delete commented codes as they were refactored

Reply:
okay deleted